README.md

```diff
@@ -403,6 +585,9 @@ If you are assembling your own [playbook](https://docs.ansible.com/ansible/lates
 - [Core GPFS](./roles/core)
 - [GPFS GUI](./roles/gui)
 - [GPFS Callhome](./roles/callhome)
+- [GPFS SMB](./roles/smb)
+- [GPFS NFS](./roles/nfs)
+- [GPFS SCALE_FILEAUDITLOGGING](./roles/scale_fileauditlogging)
```

Comment:
I guess the path should be `./roles/fileauditlogging` (no 'scale') to be consistent with the others...